src/Footer.js

```diff
@@ -0,0 +1,9 @@
+function Footer(){
+	return (
+		<footer>
+			<p>IN THE YEAR 2000</p>
```

Comment:
This is capitalized...should maybe be handled by CSS with the text-transform property.